packages/terra-menu/src/MenuItem.jsx

```diff
@@ -294,16 +294,23 @@ class MenuItem extends React.Component {
     ]);
 
     let content = textContainer;
+    const ariaLiveAttribute = markAsToggled && { 'aria-live': 'polite' };
 
     const screenReaderResponse = (
       <>
         { MenuUtils.isMac() && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.index' }, { index: index + 1, totalItems })} /> }
         { MenuUtils.isMac() && (isGroupItem || toggleable) && <VisuallyHiddenText text={markAsToggled ? intl.formatMessage({ id: 'Terra.menu.selected' }) : intl.formatMessage({ id: 'Terra.menu.unselected' })} /> }
+        {/* Adds context to Use the up and down arrows to navigate the options */}
+        { this.itemNode && this.itemNode.parentNode.getAttribute('data-submenu') === 'true' && index === 0 && totalItems !== 1 && !MenuUtils.isMac()
+          && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.navigateMenuItem' })} /> }
```

Comment:
Can we not code this such that the action are not specific to the operating system?  What is the reason that we need to check if the OS is a Mac?

Reply:
We are preventing some screen reader instructions for VO+safari and VO+chrome, for this we were using this check to prevent.

Reply:
This check won't cause any issue and we were adding the Mac logic in common file (MenuUtils.js) and imported in MenuItem.jsx.

Reply:
I am asking if we can remove the check.  Is there a reason that this logic can only be executed on a Mac?

Reply:
We have added arrow navigation instruction from submenu level for JAWS+window, that's why we were adding this check. if we remove this check screenreader is reading arrow instruction twice on main menu - [339656d](https://github.com/cerner/terra-framework/pull/1718/commits/339656d46490613a1c21e937904e476e954f6a7a)